### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by gadotroee**
I did the changed that were added in this PR: (that was opened from main) 
https://github.com/up9inc/mizu/pull/460/files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,20 +126,19 @@ $ mizu clean # mizu will continue running in cluster until clean is executed
...
 
-### Telemetry
 
-By default, mizu reports usage telemetry. It can be disabled by adding a line of `telemetry: false` in the `${HOME}/.mizu/config.yaml` file
+## Advanced Usage
 
+### Kubeconfig
 
-## Advanced Usage
+It is possible to set `KUBECONFIG` environment variable to your Kubernetes configuration. If this is not set, Mizu assumes that configuration is at `${HOME}/.kube/config`
```
Comment (IgorGov) [inline README.md:141]: Should we add here that kubeconfig can be set by config file/CLI arg?

File: README.md [hunk 2]
```diff
@@ -135,20 +126,19 @@ $ mizu clean # mizu will continue running in cluster until clean is executed
 
 ## Configuration
 
-Mizu can work with config file which should be stored in ${HOME}/.mizu/config.yaml (macOS: ~/.mizu/config.yaml) <br />
-In case no config file found, defaults will be used <br />
+Mizu can optionally work with a config file that is either provided as a CLI argument or stored in ${HOME}/.mizu/config.yaml (macOS: ~/.mizu/config.yaml) or<br />
```
Comment (IgorGov) [inline README.md:129]: Should we add the name of "CLI argument"

File: README.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ Think TCPDump and Chrome Dev Tools combined.
 ## Features
 
 - Simple and powerful CLI
-- Real-time view of all HTTP requests, REST and gRPC API calls
+- Real-time view of all HTTP requests, REST, gRPC, Redis Kafka and RabbitMQ API calls
```
Comment (mertyildiran) [inline README.md:14]:
```suggestion
- Monitoring network traffic in real-time. Supported protocols:
  - [HTTP/1.1](https://datatracker.ietf.org/doc/html/rfc2616)
  - [HTTP/2](https://datatracker.ietf.org/doc/html/rfc7540) (gRPC)
  - [AMQP](https://www.rabbitmq.com/amqp-0-9-1-reference.html) (RabbitMQ, Apache Qpid, etc.)
  - [Apache Kafka](https://kafka.apache.org/protocol)
  - [Redis](https://redis.io/topics/protocol)
```

**Review by mertyildiran — COMMENTED** [review]
Suggesting a better list of supported protocols.

File: README.md (hunk 2 above)
Comment (gadotroee) [reply README.md]: added

**Review by IgorGov — APPROVED** [review]

**Pull request merged**